cep-6.md

```diff
@@ -0,0 +1,233 @@
+
+### When will this message be shown?
+
+The notification message will appear while running the following commands:
+
+- `create`
+- `env create`
+- `env update`
+- `install`
+- `search`
+- `update`
+
+The reasoning behind this decision is that the above commands all retrieve `repodata.json` from the configured
+channels. Simply adding another file request (which is very small and immediately cached) would not add much more
+overhead to the execution of these commands. Additionally, these are commands where our users already expect network
+traffic to occur, so the requirement for having an active internet connection is assumed.
+
+Here's an example of how a notification message may appear while running the `conda create` command:
+
+```
+$ conda create -n geopandas geopandas
+Collecting package metadata (repodata.json): done
+Solving environment: done
+
+## Package Plan ##
+
+  environment location: /Users/username/opt/conda_x86_64/envs/geopandas
+
+  added / updated specs:
+    - geopandas
+
+... (output truncated)
+
+Channel (defaults) notices [info]:
+
+  Here is a message to the user
+  Here is a link they could click: https://example.com/link-name
+  To see the message again, run `conda notices`
+
+End of channel notices
+```
+
+### How else can our users access this message?
+
+Additionally, because our users may wish to see this message on demand, we will add a new sub-command called `notices`. 
+The following are a couple examples to show exactly how it would function:
+
+**Basic usage:** grabs notices for all current channels:
+
+```
+$ conda notices
+
+Channel (defaults) notices [info]:
+This is a test message. It is not very long, and could have a link to a longer post:
+https://example.com/short-link
+
+Channel (conda-forge) notices [info]:
+Here is another message. It could have info about the latest happenings or blog posts from conda-forge:
+https://conda-forge.org/
+```
+
+**Show a single channel:** grabs notices for a single channel:
+
+```
+$ conda notices -c defaults
+
+Channel (defaults) notices [info]:
+This is a test message. It is not very long, and could have a link to a longer post:
+https://example.com/short-link
+```
+
+### What file format will this message have, and what will it contain?
+
+The notification message will be in the JSON file format. This will allow us to not only store the message itself but 
+also metadata about the message, including information about how often the client should display the message (more on 
+this in the next section).
+
+Here's an example of the `notices.json` file which will be stored in the root of the channel directory structure.
+
+```json
+{
+  "notices": [
+    {
+      "id": "1cd1d8e5-d96c-42d1-9c29-e8120ad80823",
+      "message": "Here is an example message that will be displayed to users",
+      "level": "info",
+      "created_at": "2022-04-26T11:50:34+00:00",
+      "expiry": 604800,
+      "interval": 604800
```

Comment:
```suggestion
      "expiry": 604800,
      "interval": 604800,
      
      "lang": "en"
```

Do you think it would be appropriate to add a `lang` attribute here to denote the language of the message in preparation for a future in which a server could respond in a user's preferred terminal language?